sdk/helper/keysutil/policy.go

```diff
@@ -1025,7 +1025,7 @@ func (p *Policy) HMACKey(version int) ([]byte, error) {
 	return keyEntry.HMACKey, nil
 }
 
-func (p *Policy) Sign(ver int, context, input []byte, hashAlgorithm HashType, sigAlgorithm string, marshaling MarshalingType) (*SigningResult, error) {
+func (p *Policy) Sign(ver int, context, input []byte, hashAlgorithm HashType, sigAlgorithm string, saltLength string, marshaling MarshalingType) (*SigningResult, error) {
```

Comment:
Since this is part of Vault's SDK, could we keep a backwards compatible signature that passes in the `PSSSaltLengthAuto` constant? Same would go for `VerifySignature` below.